lib/engine/step/buy_sell_par_shares.rb

```diff
@@ -281,7 +281,9 @@ def activate_program_buy_shares(entity, program)
         end
         share = corporation.ipo_shares.first
         if can_buy?(entity, share.to_bundle)
-          [Action::BuyShares.new(entity, shares: share)]
+          action = Action::BuyShares.new(entity, shares: share)
+          action.user = 'programmed'
```

Comment:
can we do this another way?
like programmed = true

Reply:
Sure, do you want the JSON to change then?

Reply:
i wonder if this is necessary at all. i think people might not want it to be any different from a normal action. thoughts on just having it appear as an instant user action?

Reply:
It feels like it might confuse people since it's instant

Reply:
i think it would be fine, thoughts about just trying that for now until people complain?

Reply:
fair enough, will remove until someone complains